core/storage/question/gae_models.py

```diff
@@ -670,6 +670,8 @@ class QuestionSummaryModel(base_models.BaseModel):
         indexed=True, required=True)
     # The html content for the question.
     question_content = ndb.TextProperty(indexed=False, required=True)
+    # The ID of the interaction.
+    interaction_id = ndb.TextProperty(indexed=False, required=True)
```

Comment:
Use StringProperty here and indexed=True. TextProperty can't be indexed and we may well want to query on this field.

Reply:
Yes, done.